modules/@apostrophecms/notification/ui/apos/apps/ApostropheNotification.js

```diff
@@ -121,6 +122,7 @@ export default function() {
           :key="notification._id"
           :label="notification.message"
           :type="notification.type"
+          :icon="notification.icon"
```

Comment:
Do we need icon in addition to type? Should we just map types to icons or something? Is this part of Stu's design?

Reply:
they are passed in separately right now. The current implementation increases flexibility, but I'd be happy for it to be more limited to simplify the implementation. Out of scope of this PR, though.